include/xeus-sqlite/xeus_sqlite_interpreter.hpp

```diff
@@ -49,12 +46,15 @@ class XEUS_SQLITE_API interpreter : public xeus::xinterpreter
     nl::json kernel_info_request_impl() override;
     void shutdown_request_impl() override;
 
+    std::string sanitize_string(const std::string& code);
     std::vector<std::string> tokenizer(const std::string& code);
     bool is_magic(std::vector<std::string>& tokenized_code);
```

Comment:
still not a `const std::vector<std::string>>&` :)

the code looks much cleaner now! huge improvement :+1:

Reply:
yup

Reply:
I replied in a previous comment that I can't because I'm actually modifying the string here, so it's easier to manipulate later.
I can create a new function to do that if you think it will make the code cleaner.

Reply:
Ok fine, then keep it as it is :)